src/sass/patterns/_disclosure.scss

```diff
@@ -0,0 +1,42 @@
+// The Details disclosure element
+// ------------------------------
+
+details {
+  --link: var(--fgcolor);
+  --link-focus: var(--fgcolor);
+
+  background-color: var(--status-result-bg, var(--bgcolor));
+  border-radius: var(--border-radius);
+  color: var(--status-result-fg, var(--fgcolor));
+}
+
+summary {
+  cursor: pointer;
+  font-weight: bold;
+
+  // Hides marker some browsers add
+  &::marker {
+    content: none;
+  }
+
+  // Hides marker webkit adds
+  &::-webkit-details-marker {
+    display: none;
+  }
+
+  &::before {
+    border-color: transparent transparent transparent currentColor;
+    border-style: solid;
+    border-width: var(--triangle-width) 0 var(--triangle-width)
+    var(--triangle-height);
+    content: '';
+    grid-column: 2;
+    grid-row: 1;
+    transform: rotate(0deg);
+    transition: all 1s ease-out 100ms;
+
+    [open] & {
+      transform: rotate(90deg);
+    }
```

Comment:
@mirisuzanne @dvdherron Is there a bug in Safari that doesn't allow this before pseudo element to transition when we add the [open] attribute? I am able to see the transitions if I just toggle the properties (see screen recording) but not when I click to open/close it. I made it 1s for testing purposes so it is obvious. 




https://github.com/oddbird/oddcontrast/assets/1581694/7eaba154-ae60-4728-b9a3-b1a81206415d

Reply:
I tried searching for a bug related to this but haven't been able to find anything just yet. I know it shouldn't be needed but does adding a vendor prefix change anything @stacyk ?

Reply:
I just tried the vendor prefix and I couldn't find anything either. I found this: https://bugs.webkit.org/show_bug.cgi?id=213349 

So it looks like it was a bug, then it was fixed, and then it came back in version 16. I don't know if it is helpful to send them our use case once this is merged?